app/models/comment.rb

```diff
@@ -3,7 +3,7 @@ class Comment < ActiveRecord::Base
   belongs_to :user
   validates :body, :presence => :true
 
-  default_scope order: 'comments.created_at DESC'
+  default_scope { order('comments.created_at DESC')}
```

Comment:
Why did you change the coding style here? It doesn't match the coding style in the rest of the project.